v-next/hardhat/src/internal/builtin-plugins/network-manager/network-manager.ts

```diff
@@ -204,6 +214,10 @@ export class NetworkManagerImplementation implements NetworkManager {
             chainType: resolvedChainType as ChainType,
           },
           jsonRpcRequestWrapper,
+          tracingConfig: {
```

Comment:
have you seen any performance hit after adding this? I thought it would be disabled by default and enabled through config.

Reply:
It's meant to be enabled by default, so if there's any performance issue we should address it.

Right now it will have a small performance problem, which is being address in EDR. Note that we are parsing and transforming the build infos, just to pass it to EDR.

In the near future we'll just pass the `Uint8Array` read from the file, directly.